README.md

```diff
@@ -1,10 +1,10 @@
-![Django Backend](https://github.com/Zorroa/scrounger/workflows/Django%20Backend/badge.svg) ![React Frontend](https://github.com/Zorroa/scrounger/workflows/React%20Frontend/badge.svg)
+![Django Backend](https://github.com/TheBoonAI/scrounger/workflows/Django%20Backend/badge.svg) ![React Frontend](https://github.com/TheBoonAI/scrounger/workflows/React%20Frontend/badge.svg)
```

Comment:
"The" Boon? I thought it was just "Boon AI"

Reply:
I mean, look at the url on the current webpage where we are having this exchange 😛 

I assume "BoonAI" was not available when @danny-zorroa made the change (@danny-boonai?)